api_spec.yaml

```diff
@@ -809,30 +810,22 @@ paths:
         content:
           application/json:
             schema:
-              oneOf:
-                - type: object
-                  properties:
-                    redirectUris:
-                      type: array
-                      items:
-                        $ref: '#/components/schemas/oauth2RedirectUri'
-                    name:
-                      type: string
-                      example: asdf
-                    scopes:
-                      type: array
-                      items:
-                        $ref: '#/components/schemas/oauth2Scope'
-                  required:
-                    - redirectUris
-                    - name
-                - type: object
-                  properties:
-                    status:
-                      type: string
-                      enum: ["UNKNOWN_SCOPE"]
-                    scope:
-                      $ref: '#/components/schemas/oauth2Scope'
+              type: object
+              properties:
+                redirectUris:
+                  type: array
+                  items:
+                    $ref: '#/components/schemas/oauth2RedirectUri'
+                name:
+                  type: string
+                  example: 'developement'
+                scopes:
+                  type: array
+                  items:
+                    $ref: '#/components/schemas/oauth2Scope'
+              required:
+                - redirectUris
+                - name
```

Comment:
in the request add an optional boolean for `addDefaultSuperTokensScopes` (default value is true). 

Change the flow diagram accordingly to take this boolean into account.

Reply:
default scopes start with `supertokens:...`

Reply:
changes made to include :
- clients with no scopes
- clients with default scopes
- clients with default scopes as well as existing user-defined scopes